source/views/PhantomTimeView.mc

```diff
@@ -45,6 +45,9 @@ class PhantomTimeView extends Components.TimeView {
 
     private function updatePatterntTrans() as Void {
         self._patternTrans = SettingsModule.getValue(SettingType.DOT_HOUR_TRANS);
+        if (!GlobalKeys.CAN_CREATE_COLOR && self._patternTrans > 0) {
```

Comment:
This construct is unnecessary because null processing is already present

Reply:
If this condition is not needed, the language [Pattern transparency(Older devices[eg.fr245/745/945] cannot support 50% and 75%. Even if you set it to 50% or 75%, it will be used as 100%.)] in settings.xml also needs to be corrected.I need your advice :)